### DISCUSSION TITLE
Hover ideas !

### DISCUSSION THREAD

**Discussion by lucasbcgeo** [Akifyss/obsidian-border#204] [Ideas]
First of all, congratulations on your work once again!

My suggestions are based on the table of contents on the right side of this link here:

https://danielorodriguez.com/obsidian-modal-form/#inline-forms

When we hover over an item in the table of contents, in addition to the scrollbar gaining a highlight color, the item also gains this same highlight color.

Second suggestion: as we progress through reading the content, the items in the table of contents that we have already read or that have been left behind become colorless or with low opacity.

Third suggestion that will match very well with something that already exists in your theme.

The page that is selected can be highlighted, right? Great feature, my suggestion is to create a tool that when you hover over another window/tab, that window gains a highlight color around it different (from the window that is selected) while the mouse is over that window/tab.

Third and final suggestion: differentiate or add opacity for when the active tab is in use or not. If it is not in use but still selected, make this color a bit weaker.

As someone with a high degree of ADHD, this would be phenomenal because we wouldn't get lost among so many windows, hehehe. And it's not a critique of your theme. Quite the contrary, I had made a snippet of almost 2k lines (with the help of ChatGPT, unfortanetly i am not a dev) for another theme, but when I tried yours, I abandoned it and only kept 30 lines because I found yours to be that good.

Congrats one more time! Nice job!

**Comment by Akifyss** [comment]
Hello, I just checked this website and all these features are great. However, I'm afraid that it's not possible to determine the state of this element solely through CSS, so I won't be able to implement them.  Sorry for that.
